habito/habito.py

```diff
@@ -103,9 +115,18 @@ def list():
 
             if quanta >= habit.quantum:
                 column_text = u'\u2713'
-
-            habit_row.append("{0} ({1})".format(column_text, quanta))
-
+                if in_streak:
```

Comment:
Should we store `streak` in a model, it can be computed on every check-in? It appears we're doing too many things on the fly. Since the program is growing complex now, we can start drawing line between view and model.

May be create a `AchievementsModel`. Each habit has an entry in the model, the columns are the achievements unlocked by user. `Streak` can be a column for instance. Thoughts?